.azure-pipelines/templates/jobs/standard-tests-jobs.yml

```diff
@@ -58,9 +58,9 @@ jobs:
         apache-compat:
           IMAGE_NAME: ubuntu-18.04
           TOXENV: apache_compat
-        le-auto-xenial:
+        le-auto-centos6:
```

Comment:
Why are we running this in the standard tests rather than extended now?

Reply:
I did it to replace `le-auto-xenial` which we were previously running here. That test was:

1. Running a test on certbot-auto as part of PRs rather than doing it all nightly.
2. Running `tests/modification-check.py` to make sure that certbot-auto has been built and the copies in the root of the repo were not improperly modified. The latter is especially important because https://dl.eff.org/certbot-auto pulls from the files in the repo root. I added running this test as part of `le-auto-centos6` in the `tox.ini` changes in this PR.